webroot/adm/oncall/generate-cstg-keypair.html

```diff
@@ -0,0 +1,650 @@
+                                    <button type="button" aria-controls="createKeyPairRegion" aria-expanded="false"
+                                        id="createClient" data-bs-parent="#siteParent" class="btn btn-secondary group"
+                                        data-bs-toggle="collapse" data-bs-target="#createKeyPairRegion"
+                                        role="button">Create keypair</button>
+                                </div>
+                            </div>
+                            <div id="siteParent" class="col">
+                                <div class="row collapse mt-3" id="createSiteRegion" data-bs-parent="#siteParent">
+                                    <form id="createSiteForm">
+                                        <div class="card card-body">
+                                            <div class="form-group">
+                                                <h5>Create a new site</h5>
+
+                                                <label for="createSite-workflow-name">Name:</label>
+                                                <input type="text" class="form-control" id="createSite-workflow-name"
+                                                    name="createSite-workflow-name">
+                                                <div class="py-2">
+                                                    <label for="createSite-workflow-domainNames">Domain Names
+                                                        (comma-separated list):</label>
+                                                    <input type="text" class="form-control"
+                                                        id="createSite-workflow-domainNames"
+                                                        name="createSite-workflow-domainNames">
+
+                                                </div>
+                                                <div class="py-2">
+                                                    <label for="createSite-workflow-description">Description
+                                                        (Optional):</label>
+                                                    <input type="text" id="createSite-workflow-description"
+                                                        class="form-control" name="createSite-workflow-description">
+                                                </div>
+                                                <div class="form-check py-2">
+                                                    <input type="checkbox"
+                                                        id="createSite-workflow-clientTypes-PUBLISHER"
+                                                        class="form-check-input"
+                                                        name="createSite-workflow-clientTypes[]" value="PUBLISHER">
+                                                    <label for="createSite-workflow-clientTypes-PUBLISHER">Is this
```

Comment:
nit: could we please put this string on one line? I was trying to search for it but was proving difficult to find.

Reply:
I'm also not sure exactly how accurate this statement is - I believe advertisers can use it too.

Reply:
That's correct. Btw please see our CSTG How-to page as advertisers should generally go via the portal rather than us doing it in Admin.